presidio-anonymizer/cmd/presidio-anonymizer/anonymizer/anonymizer.go

```diff
@@ -11,6 +12,33 @@ import (
 
 type sortedResults []*types.AnalyzeResult
 
+// anonymizeSingleResult anonymize a single analyze result
+func anonymizeSingleResult(result *types.AnalyzeResult, transformations []*types.FieldTypeTransformation, text string) (bool, string, error) {
+	var err error
+	var newtext string
+	for _, transformation := range transformations {
+		if transformation.Fields == nil {
+			newtext, err = transformField(transformation.Transformation, result, text)
```

Comment:
extract this 4 lines to a method - you're doing exactly the same here and in line 30

Reply:
👍